apps/notifications/src/standalone/index.js

```diff
@@ -100,6 +100,13 @@ const init = () => {
 
 				isShowing = showing;
 				refresh();
+
+				// If we're in a WP-Desktop context, window.electron will
+				// be set. Let the app know that the notifications panel
+				// has been toggled.
+				if ( window && window.parent ) {
```

Comment:
Based on the comment is this line supposed to be `if ( window && window.electron )`?

Reply:
Yeah the comment would have to be updated.  I've been trying a few different things and forgot to update the comment each time. In this particular case I was attempting to use `postMessage` to communicate from the notification panel's iframe (since the iframe's ability to access the parent window's properties may be restricted).

Unfortunately this isn't working yet, although hopefully I'm headed in the right direction. Also as an FYI testing this requires sandboxing the WP notifications widget from this PR (so this can't work if simply running Calypso as localhost).